draft-ietf-httpbis-cache-digest.md

```diff
@@ -255,25 +315,39 @@ Servers MUST ignore CACHE_DIGEST frames sent on a stream other than 0.
 
 ### Querying the Digest for a Value {#querying}
 
-Given:
+Given the following inputs:
 
-* `digest-value`, an array of bits
-* `URL`, an array of characters
-* `ETag`, an array of characters
-* `validators`, a boolean
+* `URL` a string corresponding to the Effective Request URI ({{RFC7230}}, Section 5.5) of a cached response {{RFC7234}}.
+* `ETag` a string corresponding to the entity-tag {{RFC7232}} if a cached response {{RFC7234}} (if the ETag is available; otherwise, null).
```

Comment:
From an implementor's perspective, it would help me to understand this if examples were provided.

Reply:
Examples for URL and ETag? Or something else?

Reply:
ETag

Btw just noticed a typo on line 321: `of a cached response`